jbpm-wb-human-tasks/jbpm-wb-human-tasks-client/src/test/java/org/jbpm/workbench/ht/client/editors/taskslist/TaskAdminListViewImplTest.java

```diff
@@ -54,11 +77,57 @@ public AbstractTaskListPresenter getPresenter() {
         return Arrays.asList(TAB_SEARCH,
                              TAB_ADMIN);
     }
+    
+    @Override
+    public List<String> getExpectedInitialColumns() {
```

Comment:
For consistency reasons, this method should be overridden in _TaskListViewImplTest_ too, by copying over the implementation from _AbstractTaskListViewTest_ and leaving the method also abstract in _AbstractTaskListViewTest_ .